spec/models/blog_spec.rb

```diff
@@ -0,0 +1,48 @@
+require 'rails_helper'
+
+RSpec.describe Blog, type: :model do
+        
+  #Delete all blogs in ActiveRecord
+  Blog.delete_all
+
+  it "has 0 blogs at the start" do
+      expect(Blog.count).to eq(0)
+  end
+
+  it "has 1 blog after creating" do
+      @blog = Blog.create
+      expect(Blog.count).to eq(1)
+      @blog.destroy
+  end
+
+  ###Test fails - I get [2], so I need to reset id's when destroying
+  #it "has blog id of 1 after creation" do
+  #    @blog = Blog.create
+  #    expect(Blog.ids). to eq([1])
+  #    @blog.destroy
+  #end
+
+  it "has 0 blogs after deletion of the only one" do
+      @blog = Blog.create
+      expect(Blog.count).to eq(1)
+      @blog.destroy
+      expect(Blog.count). to eq(0)
+  end
+
+  it "has 2 blogs of differrent id's" do
+      #create an array of 2 different blogs
+      @blog_arr = [Blog.create({title:'Blog34', id:34}),
+               Blog.create({title:'Blog56', id:56})]
+      expect(Blog.count).to eq(2)
+
+      #Uncomment to see that Blogs.ids is an array [34,56]
+      #print(Blog.ids)
+      
+      #could also do a for loop
+      expect(Blog.ids). to eq([@blog_arr[0].id, @blog_arr[1].id])
```

Comment:
Wow, I didn't actually know about the `ids` method. Neat. 

Usually I'd do `Blog.pluck(:id)`. You can use the `pluck` method to get an array of any attribute (or attributes), which is sometimes useful, especially on occasions where your models have lots and lots of fields but you only need one or two of them, i.e. you don't want to load all the records into memory.

Reply:
Thanks. I got it from a combination of thinking to get an id from the Blog model (object), and then when id didn't work the error on the console when running `rake rspec` suggested `ids`.

Cool, I like that, and seems useful for the case you mention.